Examples/Tests/load_external_field/analysis_time_scaling.py

```
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# Compare B(component) scaling between two plotfiles:
+# checks that BN/B0 ~= expected_ratio on cells where |B0| > min_abs
+
+import argparse
+
+import numpy as np
+
+
+def load_plotfile_arrays(path, comp):
+    # Prefer AMReX's native plotfile reader; fall back to yt.
+    try:
+        from amrex import plotfile as amr
+
+        pf = amr.PlotFile(path)
+        arr = pf.get(comp, level=0)  # numpy array (nx, ny, nz) or (nr, nz)
+        t = float(pf.time())  # seconds
+        return arr, t
+    except Exception:
```

It would be preferable that this checks for the specific exception `ImportError`. Checking for the generic exception hides other errors that could occur making debugging harder.